src/handshake.rs

```diff
@@ -323,6 +349,45 @@ impl Request {
         Ok(None)
     }
 
+    /// Get the trusted IP address of the client.
+    ///
+    /// This method will attempt to retrieve a specific forwarded IP address of the requester
+    /// in the following manner:
+    ///
+    /// If the `X-Forwarded-For` header exists, this method will return the `trusted` right most
```

Comment:
Should not it say "left most address", or I am understanding it wrong ?

Reply:
The n'th (with n=trusted) address from the right end. Usually an address farther to the left (front) can't be trusted -- it's an arbitrary value from a proxy /client farther away from the server. E.g. if I only trust my own proxy, I want the last (right most) entry.

> If a request goes through multiple proxies, the IP addresses of each successive proxy is listed. This means, the right-most IP address is the IP address of the most recent proxy and the left-most IP address is the IP address of the originating client.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-For#Directives

Reply:
Huge thanks for the explanation. Also, I must be blind, cos I did not notice `rsplit` instead of `split`. So, apologies, my bad.